src/lib/atoms/ImagePreview/ImagePreview.tsx

```diff
@@ -138,6 +206,7 @@ function ImagePreview({
                     {withModal && (
                         <div className="imagePreview__close">
                             <div className="imagePreview__divider" />
+                            {/*@ts-ignore*/}
                             <Icon type="bc-icon-close" className="imagePreview__icon" onClick={onClose} />
                         </div>
                     )}
```

Comment:
you use this expression  imageData && (
        <Magnifier
            ref={magnifierRef}
            imgUrl={imageData}
            name={name}
            zoom={2}
            showMagnifier={isMagnifierOn}
            className='imagePreview__img'
            withRotation
            withMagnifier
        />
    ); twice, you can create a separate variable and use it const DrawMagnifier = imageData && (
        <Magnifier
            ref={magnifierRef}
            imgUrl={imageData}
            name={name}
            zoom={2}
            showMagnifier={isMagnifierOn}
            className='imagePreview__img'
            withRotation
            withMagnifier
        />
    );